package.json

```diff
@@ -8,11 +8,28 @@
   },
   "license": "MIT",
   "homepage": "https://github.com/code-dot-org/dsco_#readme",
-  "scripts": {},
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
```

Comment:
[`lerna bootstrap` docs](https://github.com/lerna/lerna/tree/main/commands/bootstrap#readme)

probably should've waited to add this (it's not really needed until we have multiple components), but it's here 🤷🏻‍♀️